felix/bpf-gpl/conntrack.h

```diff
@@ -985,14 +985,27 @@ static CALI_BPF_INLINE struct calico_ct_result calico_ct_lookup(struct cali_tc_c
 				CALI_CT_DEBUG("CT RPF failed ifindex %d != %d\n",
 						src_to_dst->ifindex, ifindex);
 			}
-			/* Do not worry about packets returning from the same direction as
-			 * the outgoing packets.
-			 *
-			 * Do not check if packets are returning from the NP vxlan tunnel.
-			 */
-			if (!same_if && !ret_from_tun && !hep_rpf_check(ctx) && !CALI_F_NAT_IF && !CALI_F_LO) {
+
+			bool recheck = false;
```

Comment:
I read "recheck" as "this packet needs to be rechecked" but I think you're using it to mean "result of the recheck is ..."?  How about calling it `rpf_passed`?